fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/token/TokenFragment.java

```diff
@@ -82,7 +81,7 @@ public View onCreateView(
       Bitmap bitmap =
           QRCode.from(gson.toJson(data))
               .withSize(Constants.QR_SIDE, Constants.QR_SIDE)
-              .withColor(ActivityUtils.getQRCodeColor(getActivity()), Color.TRANSPARENT)
+              .withColor(ActivityUtils.getQRCodeColor(requireActivity()), Color.TRANSPARENT)
```

Comment:
Mh, I'm not sure about this. Where did you see the `requireContext`? Because I've scrolled a bit the changes but if I'm not mistaken I just used `requireActivity`